src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyApprenticeshipsController.cs

```diff
@@ -99,13 +99,24 @@ public async Task<IActionResult> ConfirmApprenticeships(string accountId, Guid a
         {
             var viewModel = await GetConfirmApprenticeshipViewModel(accountId, applicationId);
             if(viewModel.Apprentices.Any(a => !a.HasEligibleEmploymentStartDate))
-            {                
-                return View("NotEligibleApprenticeships", new NotEligibileViewModel(viewModel));
+            {
+                return View("NotEligibleApprenticeships", new NotEligibleViewModel(viewModel));
             }
 
             return View(viewModel);
         }
 
+        [HttpPost]
+        [Route("remove-ineligible-apprentices/{applicationId}")]
```

Comment:
Could we just re-use the confirm-apprentices endpoint?  By adding an optional querystring parameter to remove the ineligible ones?  A querystring is a filter so we would be filtering out the ineligible ones when set.

Reply:
that did cross my mind actually